content/en/tracing/other_telemetry/connect_logs_and_traces/python.md

```diff
@@ -29,8 +29,15 @@ further_reading:
 
 ### Standard library logging
 
-To correlate your [traces][1] with your logs, update your log format to include
-the required attributes from the log record and call `ddtrace.patch(logging=True)`.
+To correlate your [traces][1] with your logs, there are two steps:
+
+Step 1) Activate automatic instrumentation through any of the options below:
+- Option 1) [Library Injection][5]
+- Option 2) Import **ddtrace** then run the application with `ddtrace-run` (ie: `ddtrace-run python appname.py`)
```

Comment:
With option 2 customers should set: `DD_LOGS_INJECTION=True` otherwise `ddtrace.patch_all()` will not enable log correlation.

Reply:
Actually I am wrong.

Reply:
This will change in ddtrace 2.0. For `ddtrace<2.0`, `ddtrace-run` will call `ddtrace.patch(logging=True)`. 

To avoid confusion we should always ask customers to set this environment variable if they would like to enable this feature.

Reply:
Docs: https://ddtrace.readthedocs.io/en/latest/advanced_usage.html#patch-logging

Reply:
Great catch, I'll add that! Does this also apply to Option 1?

Reply:
Update: I just realized I did reproduce Option 1 recently and can confirm DD_LOGS_INJECTION=true needs to be applied there too